extensions/3DTILES_implicit_tiling/0.0.0/README.md

```diff
@@ -1,37 +1,849 @@
+
+Here are some examples of template URIs and files that they match:
+
+```
+== Quadtree Example ==
+Pattern: "content/${level}/${x}/${y}/tile.b3dm"
+Valid filenames: 
+- content/0/0/0/tile.b3dm
+- content/1/1/0/tile.b3dm
+- content/3/2/2/tile.b3dm
+
+== Octree Example ==
+Pattern: "content/${level}/${z}/${y}/${x}.pnts"
+Valid filenames:
+- content/0/0/0/0.pnts
+- content/1/1/1/1.pnts
+- content/3/2/1/0.pnts
+```
+
+Unless otherwise specified, template URIs are resolved relative to the tileset.json file.
+
+![Template URI](figures/template-uri.jpg)
+
+## Content
+
+**Content** is an optional 3D model associated with a single tile, as described in the [Introduction section](https://github.com/CesiumGS/3d-tiles/tree/master/specification#introduction) of the Cesium 3D Tiles 1.0 specification, with the addition of an explicit `mimeType` to reliably identify the type of content.
+
+## Subtrees
+
+**Subtrees** are fixed-sized sections of the tileset tree used for breaking tilesets into manageable pieces.
+
+Since tilesets grow exponentially with depth, storing information about every tile in a single file is not always feasible or desirable. Even if RAM is not a direct limitation, streaming large files over the network can make loading times slower. To account for this, subtrees partition the tileset structure into pieces of bounded size.
+
+![exact cover](figures/union-of-subtrees.jpg)
+
+A subtree has a fixed number of levels defined by the `subtreeLevels` property. This describes the number of distinct levels in the tree. The number of children per tile is also fixed due to the subdivision scheme. For quadtrees, there are `4` children per tile, while octrees have `8` children per tile. Taken together, a subtree has exactly enough tiles to store a full quadtree or full octree with a limited number of levels. However, each tile may or may not exist or contain content, as a tileset only stores the tiles that are necessary.
+
+![subtree anatomy](figures/subtree-anatomy.jpg)
+
+## Availability
+
+**Availability** is a boolean that defines whether a tile, content, or subtree exist in a tileset. Availability serves two purposes:
+
+1. It provides an efficient method for checking what files are present
+2. Including ths information prevents extraneous HTTP requests that would result in 404 errors.
+
+Availability takes the form of a bitstream with one bit per node in consideration. A 1 indicates that a tile/content/subtree is available at this node. Meanwhile, a 0 indicates that no tile/content/subtree is available at this node.
+
+For cases where all nodes are available or all nodes are unavailable, the `constant` property should be used instead of storing a full bitstream. For example, `constant: 0` indicates that all bits are 0 and no bitstream must be stored.
```

Comment:
A JSON example might be helpful here. `constant: 1` might be a better example because it's a more common use case for the tree to be full rather than empty, though `constant: 0` does have its use cases too, like a tree where content only shows up deeper in the tree.